settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ = True` to ensure proper handling of time zones in your application. This is especially important if your application deals with users in different time zones.